### ISSUE TITLE
How to continue session with ellama?

### ISSUE THREAD

**Issue by Crandel** [s-kostyaev/ellama#156]
When I use ellama-chat command I got this template

```
* User
<my questions>

* Ellama
<ellama answers>
```

But when I need to continue the same conversation it will be just plain text, without separation between user prompt and ellama answers. Is it possible to add this separation somehow, without manually typing `* Ellama` and `* User`? Also it is confusing my llama3.1 model, when I do this manually.

**Comment by s-kostyaev** [comment]
Just use `ellama-chat` command, it will be added automatically. If you need to use old session, just use `ellama-load-session` and after that `ellama-chat` will continue that session.

**Comment by Crandel** [comment]
I know how to continue the session, but it is continue as plain text without any separation between me as user and ellama response. I need manually add `* User` in case I want to separate question and `* Ellama` for response. Is it possible to at least insert `* Ellama` every time the answer from llm is placed in org buffer? Also will be useful to add `* User` after llm will finish answering in order to see full answer and continue chatting.

**Comment by Crandel** [comment]
Also would be great to have a youtube video with feature presentation to show the full potential of such great package.

**Comment by Crandel** [comment]
@s-kostyaev Can you please reopen this issue again?

**Comment by Crandel** [comment]
I dig around ellama code and found a solution for at least my second request

```elisp
  (defun vd/insert-ellama-prompt ()
      "Insert 'hello' at the end of the buffer."
    (interactive)
    (goto-char (point-max))
    (insert "")
    (insert "* User")
    (insert "")
    )
  (setopt ellama-chat-done-callback       'vd/insert-ellama-prompt)
```

But would be nice to have a prefix `\n* Ellama\n` before ollama stream started

**Comment by s-kostyaev** [comment]
I can't reproduce your issue. Write step by step guide what are you doing. With your callback what you will see if you call `ellama-chat` one more time?

**Comment by Crandel** [comment]
> I dig around ellama code and found a solution for at least my second request

Looks like this solution is working only once. Next ellama-complete command on the same session do not trigger callback function.

**Comment by s-kostyaev** [comment]
Why do you use `ellama-complete`? Use `ellama-chat` instead.

**Comment by Crandel** [comment]
Ok, now I understand finally how it should work. New prompt confused me, as I though it should start new session. Maybe explicitly mention this behavior in README?
